Framework/MDAlgorithms/test/ConvertMDHistoToMatrixWorkspaceTest.h

```diff
@@ -494,4 +494,101 @@ class ConvertMDHistoToMatrixWorkspaceTest : public CxxTest::TestSuite {
   }
 };
 
+/* Disabling clang format because the CXX parsser cannot read across multiple
+lines.
+ Clang formatting has therefore been disabled to prevent compiler errors.
+ When the parsser has been fixed (can read across multiple lines) the clang
+formatting
+can be enabled */
+
+// clang-format off
+class ConvertMDHistoToMatrixWorkspaceTestPerformance:public CxxTest::TestSuite {
+public:
+  static ConvertMDHistoToMatrixWorkspaceTestPerformance *createSuite() {
+    return new ConvertMDHistoToMatrixWorkspaceTestPerformance;
+  }
+  static void destroySuite(ConvertMDHistoToMatrixWorkspaceTestPerformance *suite) {
+    delete suite;
+  }
+//clang-format on
+  ConvertMDHistoToMatrixWorkspaceTestPerformance() {
+    std::vector<size_t> nonIntegr(2);
```

Comment:
This can all be done in a single line, additionally use std::array as we know the length at compile time which does not change:
`std::array<size_t> nonIntegr{0,1}` - Will create an array with values 0, 1
Then the two lines `nonIntegr[n] = x ` can be removed